lib/iris/analysis/maths.py

```diff
@@ -598,12 +623,12 @@ def log10(cube, in_place=False):
     new_dtype = _output_dtype(np.log10, cube.dtype, in_place=in_place)
     op = da.log10 if cube.has_lazy_data() else np.log10
     return _math_op_common(
-        cube, op, cube.units.log(10), new_dtype, in_place=in_place
+        cube, op, cube.units.log(10), new_dtype=new_dtype, in_place=in_place
     )
 
 
 def apply_ufunc(
-    ufunc, cube, other_cube=None, new_unit=None, new_name=None, in_place=False
+    ufunc, cube, other=None, new_unit=None, new_name=None, in_place=False
```

Comment:
This is a breaking change to the API for `apply_ufunc`, but it is completely warranted, as changing the kwarg from `other_cube` to `other`, aligns this with the rest of the `iris.analysis.maths` public API.
Consequently, there will be an appropriate entry in the `whatsnew`